### DISCUSSION TITLE
AvroSRLoadGenerator triggered for Protobuf file

### DISCUSSION THREAD

**Discussion by venkataklagisetty** [sngular/kloadgen#310] [General]
Hi
I am trying to test kafka producer using jmeter.
I am using protobuf schema , but still it triggers AvroSRLoadGenerator instead of "Proto" LoadGenerator.

Please help me with it

Here is the error:
2022-11-14 15:45:52,524 ERROR n.c.k.l.i.AvroSRLoadGenerator: Please make sure that properties data type and expression function return type are compatible with each other

**Comment by dianaGomezGar** [comment]
Hi @venkataklagisetty,  

You can open an issue with the specification of your test and we will review it. 

Cheers.
